src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Ascii.cs

```diff
@@ -0,0 +1,1244 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace System.Buffers.Text
+{
+    /// <summary>
+    /// Provides APIs for performing text-like operations over <see cref="byte"/> or
+    /// <see cref="char"/> buffers which represent ASCII text.
+    /// </summary>
+    /// <remarks>
+    /// For the purposes of this class, "ASCII" means any <see cref="byte"/> or
+    /// <see cref="char"/> in the range 0 - 127, inclusive.
+    ///
+    /// Unless otherwise specified, APIs on this class treat non-ASCII data as opaque.
+    /// This means that <see cref="Equals(ReadOnlySpan{char}, ReadOnlySpan{char})"/>
+    /// may produce a different result than <see cref="string.Equals(string?, string?, StringComparison)"/>
+    /// with <see cref="StringComparison.OrdinalIgnoreCase"/>.
```

Comment:
Nit: as long as you're providing an example, could you be explicit about what the difference might be?